### PR TITLE
[WIP][#41] AudioRecorder Component

### PR DESCRIPTION
**Pull request by jtaaa**
This is the initial design of the AudioRecorder component for recording functionality to be added to.

Fixes: #41

### CODE REVIEW

File: components/AudioRecorder/index.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Image, StyleSheet, Text, TouchableHighlight, View } from 'react-native';
+import _ from 'lodash-fp';
+
+const micIcon = require('./../../assets/ic_mic_white_24dp_2x.png');
```
Comment (katrinacrisostomo) [inline components/AudioRecorder/index.js]: wait for @FalahShazib 's branch to merge but we're going to be handling icons in a diff way :) 

she's making an Icon component that you can use in place of this

Comment (jtaaa) [reply components/AudioRecorder/index.js]: Okay sounds great!

File: components/AudioRecorder/index.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Image, StyleSheet, Text, TouchableHighlight, View } from 'react-native';
+import _ from 'lodash-fp';
```
Comment (katrinacrisostomo) [inline components/AudioRecorder/index.js]: yayy using `lodash`

**Comment by georgeke** [comment]
this is lit! Can you add some screenshots too?

**Comment by georgeke** [comment]
also since I'm working on the recording w/ Expo, I'll pull your branch push my recording changes on it

**Comment by georgeke** [comment]
we still gotta convert this to an FSC

**Comment by jtaaa** [comment]
Here's a screenshot of the component!

screenshot_20180312-220635

**Comment by georgeke** [comment]
@katrinacrisostomo added a TODO where you can add the save URI to store logic

**Review by katrinacrisostomo — APPROVED** [review]

**Pull request merged**